src/mapd-crossfilter.js

```diff
@@ -1720,6 +1721,7 @@ export function replaceRelative(sqlStr) {
           };
 
           return new Promise((resolve, reject) => {
+            cache.emptyCache()
```

Comment:
hmm, somehow you ended up with one of my commits from a branch.

Reply:
so you're saying I should remove this line @nytai ?

Reply:
Yes. If you look at the original commit, it's the single line above https://github.com/mapd/mapd-crossfilter/pull/52/files#diff-adc0a2e27d865517d83ef377a540f9bcR163

https://github.com/mapd/mapd-crossfilter/commit/cf826bf1b11f1c937be76088807192f094be8415